packages/subgraph/package.json

```diff
@@ -10,30 +10,39 @@
         "codegen": "graph codegen",
         "build": "graph build",
         "test": "truffle test --network ganache",
-        "prepare-local": "mustache config/ganache.json subgraph.template.yaml > subgraph.yaml",
+        "prepare-local": "mustache config/ganache.json test-subgraph.template.yaml > subgraph.yaml",
```

Comment:
Please also bump version to "1.0.0"

Reply:
also updated the version in `package.json`